Dockerfile

```diff
@@ -50,6 +50,7 @@ mamba install -y -n base \
     "rapids=${RAPIDS_VER}.*" \
     "dask-sql=${DASK_SQL_VER%.*}.*" \
     "python=${PYTHON_VER}.*" \
+    "cuda-python=${CUDA_VER%.*}.*" \
```

Comment:
This should be fine, but I'd like for us to add a comment somewhere that indicates this is necessary for CUDA 12 and links to https://github.com/conda-forge/cuda-python-feedstock/issues/66.